text/duration.go

```diff
@@ -0,0 +1,28 @@
+package text
+
+import (
+	"fmt"
+	"time"
+)
+
+//HumanizeDuration formats a time duration to human readable format
+//The duration in floored to the nearest unit. so 75 seconds becomes 1 minute
+//biggest unit is days (no weeks, months or years )
+func HumanizeDuration(duration time.Duration) string {
```

Comment:
do we not need a license and attribution here ?

Reply:
No. I wrote it.